envoy/auth_server/main.go

```diff
@@ -140,8 +85,60 @@ func main() {
 	// Register envoy proto server
 	envoy_auth.RegisterAuthorizationServer(grpcServer, authHandler)
 
-	fmt.Printf("Auth server starting on port %d...\n", port)
+	fmt.Printf("Auth server starting on port %d...\n", config.Port)
 	if err = grpcServer.Serve(listen); err != nil {
 		panic(err)
 	}
 }
+
+/* -------------------- Gateway Init Helpers -------------------- */
+
+// getConfigPath returns the full path to the config file
```

Comment:
This comment doesn't make any mentions of:

1. The optional override flag
2. The value of `defaultConfigPath`


Here's what I would do:
1. Call out that if the flag is set, it takes precedence
2. Provide a default as a passed in variable so the onus of tha tis on the caller, and here its just scope as a variable